specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlAzureDbForMySqlSyncTask.json

```diff
@@ -0,0 +1,478 @@
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/MigrateMySqlAzureDbForMySqlSyncTaskOutput"
+        }
+      ]
+    },
+    "MigrateMySqlAzureDbForMySqlSyncTaskOutputDatabaseError": {
+      "type": "object",
+      "x-ms-discriminator-value": "DatabaseLevelErrorOutput",
+      "properties": {
+        "errorMessage": {
+          "type": "string",
+          "description": "Error message"
+        },
+        "events": {
+          "type": "array",
+          "items": {
+            "$ref": "./TasksCommon.json#/definitions/SyncMigrationDatabaseErrorEvent"
+          },
+          "description": "List of error events."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/MigrateMySqlAzureDbForMySqlSyncTaskOutput"
+        }
+      ]
+    },
+    "MigrateMySqlAzureDbForMySqlSyncTaskOutputMigrationLevelV2": {
+      "x-ms-discriminator-value": "MigrationLevelOutput",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MigrateMySqlAzureDbForMySqlSyncTaskOutput"
+        },
+        {
+          "$ref": "./MigrateMySqlAzureDbForMySqlReplicateChangesTask.json#/definitions/MigrateMySqlAzureDbForMySqlReplicateChangesTaskOutputMigrationLevel"
+        }
+      ]
+    },
+    "MigrateMySqlAzureDbForMySqlSyncDatabaseInput": {
+      "type": "object",
+      "description": "Database specific information for MySQL to Azure Database for MySQL migration task inputs",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the database"
+        },
+        "targetDatabaseName": {
+          "type": "string",
+          "description": "Name of target database. Note: Target database will be truncated before starting migration."
+        },
+        "migrationSetting": {
+          "type": "object",
+          "description": "Migration settings which tune the migration behavior",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sourceSetting": {
+          "type": "object",
+          "description": "Source settings to tune source endpoint migration behavior",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "targetSetting": {
+          "type": "object",
+          "description": "Target settings to tune target endpoint migration behavior",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tableMap": {
+          "type": "object",
+          "description": "Mapping of source to target tables",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tablesToMigrateSchema": {
+          "type": "object",
+          "description": "Mapping of source to target tables for schema migration",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "selectedViews": {
```

Comment:
>selectedViews [](http://example.com/codeflow?start=9&length=13)

Same name and type feedback for these properties as for similarly-named properties of MigrateMySqlAzureDbForMySqlOfflineDatabaseInput.